tests/output/ConsoleInteractionTest.py

```diff
@@ -485,11 +476,14 @@ def test_print_results_for_file(self):
                 {},
                 color=False)
             self.assertEqual("""\nfilename
-|   2| line•2
+|   2| {0}
 |    | [NORMAL] SpaceConsistencyBear:
-|    | Trailing whitespace found
-""",
-                             stdout.getvalue())
+|    | {1}
```

Comment:
no need to explicitly number the placeholders, as you use them only once. `{0}` or `{1}` --> `{}`